### ISSUE TITLE
How to stop the mod from being built-in

### ISSUE THREAD

**Issue by Apollonu** [Apollounknowndev/tectonic#268]
I have a mod that extends -y and +y height, and I cannot change the mod loading order with any version above 2.0.0 because it is built-in. Is there a way for me to stop tectonic from being built in, like a json to edit or file to remove? Forge 1.20.1

**Comment by Apollounknowndev** [comment]
Reordering the mods won't fix the issue; if the mod that extends world height loads above Tectonic, you'll get vanilla terrain generation instead. You need to make manual edits to the mod files using a datapack if you want a height limit different than vanilla or the built-in `increased_height` setting.

**Comment by Apollonu** [comment]
Thats not true, Ive gotten the recent version and 2.0.0 which I am currently using to work in tandem. I need to edit alot but it does work. The only issue with the recent versions is that I cannot make them un built-in, to set automatic load order. The only mod that I have had an issue somewhat like that is lithosphere which removes ocean water.

**Comment by Apollonu** [comment]
At this point im getting told 2 different things and having 2 different issues. Tectonic above SWL locks caves from the top at -40  (config height only works for 1 value)
SWL above Tectonic removes the cave gen entirely for tectonic

**Comment by Apollonu** [comment]
I could care less about the caves, I just want to be able to make the mod not built-in

**Comment by Apollounknowndev** [comment]
You can't make the mod not built-in.
